### PR TITLE
Add support for detecting Installer VISE

### PR DESCRIPTION
**Pull request by SilasLaspada**
I wanted to add better version detection, but I currently don't see any viable way to do so.

### CODE REVIEW

File: BurnOutSharp/PackerType/InstallerVISE.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        /// <summary>
+        /// Set of all ContentMatchSets for this protection
+        /// </summary>
+        private static readonly List<ContentMatchSet> contentMatchers = new List<ContentMatchSet>
+        {
+            //TODO: Add exact version detection for Windows builds, make sure versions before 3.X are detected as well, and detect the Mac builds.
+
+            // ViseMain
+            new ContentMatchSet(
+                new ContentMatch(new byte?[] { 0x56, 0x69, 0x73, 0x65, 0x4D, 0x61, 0x69, 0x6E }, start: 0xE0A4, end: 0xE0A5),
```
Comment (mnadareski) [inline BurnOutSharp/PackerType/InstallerVISE.cs:18]: Going to admit, this range of indices seems a little iffy. Can you add to the PR description the testing you did to get to this?

File: BurnOutSharp/PackerType/InstallerVISE.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            if (!File.Exists(file))
+                return null;
+
+            using (var fs = File.OpenRead(file))
+            {
+                return Scan(scanner, fs, file);
+            }
+        }
+
+        /// <inheritdoc/>
```
Comment (mnadareski) [inline BurnOutSharp/PackerType/InstallerVISE.cs:43]: Can you add a comment here similar to Inno (if and only if there's some known extractor program OR technical specification for this packer available)?

File: README.md [hunk 3]
```diff
@@ -93,6 +93,7 @@ Below is a list of the executable packers that can be detected using this code:
 - dotFuscator
 - EXE Stealth
 - Inno Setup
+- Installer VISE
```
Comment (mnadareski) [inline README.md:96]: Is this really called "Installer VISE" and not "VISE Installer"? That's weird...

**Comment by SilasLaspada** [comment]
It is officially called "Installer VISE" everywhere I could find, the range is accurate as of the 11 test files I have, including various installers that are versions 3.1-3.7, and an installer used for a third party program, where the rest of the installers are for Installer VISE itself, and the only extraction software I could find is Universal Extractor 2.
explorer_7Vrfpi8DlG

**Review by mnadareski — APPROVED** [review]

**Pull request merged**